e2e/__tests__/multiProjectRunner.test.ts

```diff
@@ -34,6 +34,26 @@ test("--listTests doesn't duplicate the test files", () => {
   expect(stdout).toMatch('inBothProjectsTest.js');
 });
 
+test("--listTests doesn't find any tests when test path is part of absolute path", () => {
+  writeFiles(DIR, {
+    './backend/__tests__/backend.test.js': `test('test', () => {});`,
+    './frontend/__tests__/frontend.test.js': `test('test', () => {});`,
+    '.watchmanconfig': '',
+    '/project1.js': `module.exports = {rootDir: './backend', displayName: 'BACKEND'}`,
+    '/project2.js': `module.exports = {rootDir: './frontend', displayName: 'BACKEND'}`,
+    'package.json': JSON.stringify({
+      jest: {projects: ['<rootDir>/project1.js', '<rootDir>/project2.js']},
+    }),
+  });
+
+  // ignore first seperator
+  const [, , testPath] = DIR.split(path.sep);
```

Comment:
This test will fail if it happens to be located in `/jest` or something. Your test should probably do a bit more work to figure out where the files are located and run only in the case where it matters.

Reply:
Thanks for the feedback! Let me take a look at it again.